### ISSUE TITLE
Radiology text

### ISSUE THREAD

**Issue by andy-li-cv** [batmanlab/Mammo-CLIP#8]
Hi, 
I have a couple of questions:
1. Will you make the radiology texts publicly available?
2. Did u include RSNA data during pre-training as well?

**Comment by kayhan-batmanghelich** [comment]
The private dataset will stay private. We are legally responsible. I leave
the RSNA question to @shantanu.

**Comment by shantanu-ai** [comment]
Hi @andy-li-cv,
PFB the answers:
1. No. We will not release any radiology texts. We released the model checkpoints, so if you want to finetune/evaluate, you can do that. We also released the code, sample csv files similar to the datasets (**the texts are templated not actual ones**), so u can pretrain yourself.
2. No. RSNA dataset was only used for evaluation.

**Comment by andy-li-cv** [comment]
Thanks for quick answering. I have another question: How the Mammo-Factor is related to the Mammo-CLIP? What components are used in Mammo-Factor from Mammo-CLIP? Did you only use the templated texts by radiologist? Can i use the report from mammograms to localization?

**Comment by shantanu-ai** [comment]
Hi,
Mammo-Factor needs Mammo-CLIP vision and text encoders. When u try to localize a finding, u need to pass the image and the text through the vision and text encoders for the embeddings and train the contrastive loss in the paper which will align the text embeddings to the particular finding, e.g, mass, calcification etc. For the text, u can use the templated text mentioned in the appendix of the paper (in prompt.json file in the codebase) or u can use real-life radiology texts for localization.
